src/benchq/resource_estimation/_footprint_analysis.py

```diff
@@ -54,82 +55,89 @@ def estimate_cost(self) -> CostEstimate:
                 factories as fast as the factories run.
             - The bottleneck time cost is waiting for magic states.
         """
-        logical_storage = int(
+        if self.t_count == 0 and self.toffoli_count == 0:
+            raise ValueError("Circuit must contain T gates and/or Toffoli gates.")
+
+        n_logical_qubits_used_for_computation = int(
             math.ceil(
                 self.max_allocated_logical_qubits
                 * (1 + self.routing_overhead_proportion)
             )
         )
-        storage_area = logical_storage * _physical_qubits_per_logical_qubit(
-            self.logical_data_qubit_distance
+        n_physical_qubits_used_for_computation = (
+            n_logical_qubits_used_for_computation
+            * _physical_qubits_per_logical_qubit(self.logical_data_qubit_distance)
         )
-        distillation_area = (
-            self.factory_count * self.magic_state_factory.physical_qubit_footprint
+
+        n_physical_qubits_used_for_distillation = (
+            self.factory_count * self.widget.physical_qubit_footprint
         )
-        if self.t_count == 0 and self.toffoli_count == 0:
-            raise ValueError(
-                "Cannot accommodate circuits having no T gates and Toffoli gates."
-            )
 
-        rounds = int(
-            (self.toffoli_count + math.ceil(self.t_count / 2))
+        n_total_distillations = self.toffoli_count + math.ceil(self.t_count / 2)
+
+        total_distillation_cycles = int(
+            n_total_distillations
             / self.factory_count
-            * self.magic_state_factory.rounds
+            * self.widget.distillation_time_in_cycles
         )
-        distillation_failure = self.magic_state_factory.failure_rate * (
-            math.ceil(self.t_count / 2) + self.toffoli_count
+        distillation_failure = self.widget.failure_rate * n_total_distillations
+
+        V_computation = (
```

Comment:
`V_computation` is more obvious if there was an area somewhere. Right now it looks like qubits*time= volume

Reply:
The area is the qubits. Since they are all laid out in a 2d grid, they represent an area.